example/monster/listeners/on-message.js

```diff
@@ -0,0 +1,52 @@
+/**
+ *   Wechaty - https://github.com/chatie/wechaty
+ *
+ *   @copyright 2016-2017 Huan LI <zixia@zixia.net>
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+const fs = require('fs')
+
+const { MediaMessage } = require('wechaty')
+const { Misc } = require('wechaty/misc')
```

Comment:
How about to use `import { MediaMessage, Misc } from 'wechaty'` at here?

1. I'm using Node v9.5 now and it seems to support this syntax already.
1. We should import `Misc` from the module standard export.